src/rfac/rflorentz.c

```
@@ -0,0 +1,162 @@
+ *   GH/1992.10.05 - creation.
+ ********************************************************************/
+
+/*! \file
+ *  \brief Implementation for rfac_lorentz() function.
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stddef.h>
+#include <malloc.h>
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)